src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -826,6 +835,44 @@ private BoundLambda ReallyBind(NamedTypeSymbol delegateType, bool inExpressionTr
                 refKind,
                 returnType);
 
+        private LambdaSymbol? _lazyLambdaSymbol = null;
+
+        internal LambdaSymbol? LazyInProgressLambdaSymbol
```

Comment:
>LazyInProgressLambdaSymbol [](http://example.com/codeflow?start=31&length=26)

Let's give this a more specific name, perhaps `LambdaForParameterDefaultValues`.